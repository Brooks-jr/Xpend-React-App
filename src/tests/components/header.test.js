import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/header';

test('should render header component', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();

    // expect(wrapper.find('h1').text()).toBe('Xpenditure App!');
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
});