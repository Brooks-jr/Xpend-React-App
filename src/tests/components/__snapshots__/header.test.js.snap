// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header component 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__content"
    >
      <Link
        className="header__link"
        replace={false}
        to="/dashboard"
      >
        <h1
          className="header__link-text"
        >
          <span>
            X
          </span>
          pend
        </h1>
      </Link>
      <button
        className="button button--link"
        onClick={[Function]}
      >
        Sign Out
      </button>
    </div>
  </div>
</header>
`;
